bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -146,6 +154,9 @@ public Configuration toHBaseConfig() {
     config.set(BigtableOptionsFactory.PROJECT_ID_KEY, getProjectId());
     config.set(BigtableOptionsFactory.ZONE_KEY, getZoneId());
     config.set(BigtableOptionsFactory.CLUSTER_KEY, getClusterId());
+    for (Entry<String, String> entry : additionalConfiguration.entrySet()) {
```

Comment:
If additionalConfiguration contains config for CLUSTER_KEY let say, it overrides "config.set(BigtableOptionsFactory.CLUSTER_KEY, getClusterId());" Please document the order.